lib/avromatic/model/logical_types.rb

```diff
@@ -0,0 +1,16 @@
+module Avromatic
+  module Model
+    module LogicalTypes
+
+      LOGICAL_TYPE_MAP = {
+        'date' => Date,
+        'timestamp-micros' => Time,
+        'timestamp-millis' => Time
+      }.freeze
+
+      def self.value_class(logical_type)
+        LOGICAL_TYPE_MAP.fetch(logical_type, nil)
```

Comment:
For some reason when I wrote this this felt like it captured more explicitly that its okay for a logical type to not be present, and we're expecting to handle nil.